endpoints.json

```diff
@@ -77,5 +77,23 @@
         ]
       }
     }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "post a comment on a given article",
```

Comment:
You'll thank me when we get to front end - but i would document here what the post body needs to look like i.e. what keys they need to send